src/View/Helper/FormHelper.php

```diff
@@ -1707,13 +1707,21 @@ public function postLink($title, $url = null, array $options = [])
         unset($options['block']);
 
         $url = '#';
-        $onClick = 'document.' . $formName . '.submit();';
-        if ($confirmMessage) {
-            $options['onclick'] = $this->_confirm($confirmMessage, $onClick, '', $options);
+
+        if(!empty($options['class'])) {
```

Comment:
I'm sorry, but I am not familiar with the abbreviation CS here .. you mean code style with that? The space between ```if``` and ```(``` is meant, right?